### DISCUSSION TITLE
Email notifications not being sent

### DISCUSSION THREAD

**Discussion by tbrlpld** [community/community#36599] [Other Feature Feedback, Questions, & Ideas] [answered]
I could not find another way to contact support, so I leave it here... (please let me know if there is a better way...)

I am currently not receiving email notifications for @mentions and some other options. I do receive some email notifications but not for that case. 

Is there an issue with email notifications on Github's side or am I doing something wrong? 

Here I my settings:
<img width="876" alt="image" src="https://user-images.githubusercontent.com/24797493/196560261-54745fd9-eb53-4195-a5ad-0a24227937b2.png">

**Comment by radar-blip** [comment, marked as answer]
For account related issues such as yours, see here: [Support page](https://support.github.com/contact/account)

Note: Your settings look correct to me, same as mine.

**Reply by tbrlpld** [reply]
Thanks @linuxlawson

**Reply by radar-blip** [reply]
You're welcome. Hope you get it figured out!

**Reply by tbrlpld** [reply]
Thanks. Still waiting for a response.

**Comment by ossanna16** [comment]
Hey there @tbrlpld and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by ran-codes** [comment]
same problem for someone in my org. Any luck?

**Comment by tbrlpld** [comment]
Hi, yes I was somewhat able to solve this with help from support linked above. It turned out there are two connected settings. See the response below:

> Also, on further investigation, Engineering has confirmed this is expected behavior - in order to receive email notifications for @mentions in comments, the `Comments on Issues and Pull Requests` option will need to be checked. Having this disabled will prevent email notifications from being sent for @mentions in comments but will still trigger an email for mentions in the description of the issue or pull request.
> 
> I think our documentation could be more clear on this, and see we've received similar feedback in the past. We have an internal issue open regarding this confusion and I've added your experience to that issue.

**Reply by ran-codes** [reply]
thank you for sharing!
